### PR TITLE
[SET-25] API Testing

### PR DESCRIPTION
**Pull request by dallascrichmond**
## 🎯 Summary
- Added and configured Jest and supertest for testing the api
- Updated the delete operation in the inquiry router and controller
- Updated CRUD operations in the inquiry-controller
- Wrote some initial integration tests for the health, inquiry and user routes
- Added test command to route package.json

[SET-25](https://citz-imb.atlassian.net/jira/software/c/projects/SET/boards/51?selectedIssue=SET-25)

## 🔰 Checklist

- [x] I have read and agree with the following checklist and am following the guidelines in our [Code of Conduct](/CODE_OF_CONDUCT.md) document.

> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - I have consulted with the team if introducing a new dependency.
> - My changes generate no new warnings.

### CODE REVIEW

File: api/controllers/inquiry-controller.ts [hunk 1]
```diff
@@ -48,15 +48,17 @@ export const updateInquiry = async (req: Request, res: Response) => {
  * @author dallascrichmond
  */
 export const deleteInquiry = async (req: Request, res: Response) => {
+    const { id } = req.params;
+    const inquiryID: number = +id;
```
Comment (dbarkowsky) [inline api/controllers/inquiry-controller.ts:52]: What happens here if id is undefined?

**Review by dbarkowsky — APPROVED** [review]
One comment to think about. Might be worth writing a test for that. Otherwise looks good.

File: api/controllers/inquiry-controller.ts (hunk 1 above)
Comment (dallascrichmond) [reply api/controllers/inquiry-controller.ts:52]: Good point! I'll add a test for that/ make alterations if needed!

Comment (dallascrichmond) [reply api/controllers/inquiry-controller.ts:52]: @dbarkowsky I added two new tests for invalid IDs when deleting and updating inquiries

**Review by SDToews — APPROVED** [review]
code looks good, but I have to bring up all the containers and then login to the application in order to have the tests run successfully.  I don't think this would make it past the automated testing in the pipeline (though currently it looks like it is skipping it)

**Pull request merged**